sources/engine/Stride.UI/Controls/ScrollViewer.cs

```diff
@@ -847,25 +847,19 @@ protected override void OnPreviewTouchMove(TouchEventArgs args)
                 args.Handled = true;
         }
 
-        private static void RaiseLeaveTouchEventToHierarchyChildren(UIElement parent, TouchEventArgs args)
+        private static void RaiseLeavePointerEventToHierarchyChildren(UIElement parent, PointerEventArgs args)
         {
             if (parent == null)
                 return;
 
-            var argsCopy = new TouchEventArgs
-            {
-                Action = args.Action,
-                ScreenPosition = args.ScreenPosition,
-                ScreenTranslation = args.ScreenTranslation,
-                Timestamp = args.Timestamp
-            };
```

Comment:
Yeah it is mutated when it is raised, this includes, setting the `Source` property, the `RoutedEvent` property, the `IsBeingRouted` property, and a user can set the `Handled` property. Can't just reset the properties before/after because it is still passed on to the children. Also seems more error prone. 

It seems worth while to intorduce event arg pooling to avoid a lot of the allocation with the events. But that seemed better to do in a seperate PR later.